icepyx/core/temporal.py

```diff
@@ -1,12 +1,13 @@
-import datetime as dt
-import warnings
-
 """
 Helper functions for validation of dates
 """
 
+import datetime as dt
+from typing import Union
+import warnings
+
```

Comment:
I know this reordering is a ruff thing, but it doesn't make sense in this case because the long comment isn't describing the whole module, only the first portion of it, while the imports apply to the whole thing.

Reply:
Ah, this was my mistake then, I mistook that for a module-level docstring that had gotten moved to the wrong place over time. This makes much more sense. Are you OK with me changing this to a `#` comment? The fact that it was in triple-quoted style is what stimulated my jump to that conclusion!

Reply:
7751e26